ros/finalarm_description/launch/gazebo.launch

```diff
@@ -1,20 +1,31 @@
 <launch>
-  <include
-    file="$(find gazebo_ros)/launch/empty_world.launch" />
+  <include file="$(find gazebo_ros)/launch/empty_world.launch" />
+  <arg
+    name="model"
+    default="$(find finalarm_description)/urdf/finalarm_description.urdf.xacro"
+  />
+  <param
+    name="robot_description"
+    command="$(find xacro)/xacro $(arg model) use_gazebo:=true"
```

Comment:
I've defaulted to using xacro where-ever possible. There's some backwards compatibility baked in, but if it's broken on other code, they should instead pin the repository to an older version or upgrade.